var/spack/repos/builtin/packages/gpi-space/package.py

```diff
@@ -53,7 +55,8 @@ class GpiSpace(CMakePackage):
         when="+iml",
     )
 
-    depends_on("cmake@3.15.0:", type="build")
+    depends_on("cmake@3.15.0:", type="build", when="@:22.12")
```

Comment:
```suggestion
    depends_on("cmake@3.15.0:", type="build")
```

Reply:
Thanks, I didn't know adding the constraint in such a way was sufficient.